README.md

```diff
@@ -0,0 +1,13 @@
+📗 Hello Microverse
+
+Hello Microvese is a webpage which says "Hello Microverse"
+
+🛠 Built With
+
+HTML
+
+CSS
+
+Key Features
+
+Well... it says HELLO! what more do you want?
```

Comment:
Well done, @matee-safi :+1: 
- [ ] I see that you are making great efforts here but please as the @abdulhamiid suggested, it will be a lot better if you can work some on your ReadMe file. Check [this template](https://github.com/microverseinc/readme-template) for more detail. I'm rooting for you, Champ! :muscle: :muscle: